src/main/resources/marc4bibframe.yml

```diff
@@ -9,7 +9,6 @@ fieldRules:
       append: true
       edges:
         - types: CATEGORY
-          parent: WORK
```

Comment:
"parent" is unnecessary here. As this block is under an "edges" block, we already know what is the parent.